common/djangoapps/third_party_auth/tests/specs/base.py

```diff
@@ -65,14 +70,18 @@ def assert_register_response_in_pipeline_looks_correct(self, response, pipeline_
         # Check that the correct provider was selected.
         self.assertContains(
             response,
-            u'successfully signed in with <strong>%s</strong>' % self.provider.name,
```

Comment:
**Note to reviewers:** Since the new implementation [presents this string via Javascript](https://github.com/edx/edx-platform/blob/91e5ab93dee8cacf632b3328de657e9f1a2fd023/lms/static/js/student_account/views/RegisterView.js#L478) in Backbone, we check for the field value in the response instead.